C01/project-metadata/C01-ProjectId612-metadata.json

```diff
@@ -69,12 +69,44 @@
       }
     },
     "type": "Feature",
+    "place_name": "Taita–Taveta, Kenya",
     "geometry": {
       "type": "Point",
       "coordinates": [
         38.802647,
         -3.910022
       ]
     }
-  }
+  },
+  "regen:previewPhoto": {
+    "@type": "schema:URL",
+    "@value": "https://regen-registry.s3.amazonaws.com/projects/kasigau/kasigau.png"
+  },
+  "regen:galleryPhotos": {
```

Comment:
i think this is a field we need to add to the C01 project schema..

Reply:
ditto